mmv1/products/compute/RouterBgpPeer.yaml

```diff
@@ -282,3 +284,34 @@ properties:
       such as Next Gen Firewalls, Virtual Routers, or Router Appliances.
       The VM instance must be located in zones contained in the same region as
       this Cloud Router. The VM instance is the peer side of the BGP session.
+  - !ruby/object:Api::Type::Boolean
+    name: 'enableIpv6'
+    description: |
+      Enable IPv6 traffic over BGP Peer. If not specified, it is disabled by default.
+    default_value: false
+  - !ruby/object:Api::Type::String
+    name: 'ipv6NexthopAddress'
+    description: |
+      IPv6 address of the interface inside Google Cloud Platform.
+      The address must be in the range 2600:2d00:0:2::/64 or 2600:2d00:0:3::/64.
+      If you do not specify the next hop addresses, Google Cloud automatically
+      assigns unused addresses from the 2600:2d00:0:2::/64 or 2600:2d00:0:3::/64 range for you.
+    default_from_api: true
+    required_with:
+      - enableIpv6
+    validation: !ruby/object:Provider::Terraform::Validation
+      function: 'validateIpAddress'
+    diff_suppress_func: ipv6RepresentationDiffSuppress
+  - !ruby/object:Api::Type::String
+    name: 'peerIpv6NexthopAddress'
+    description: |
+      IPv6 address of the BGP interface outside Google Cloud Platform.
+      The address must be in the range 2600:2d00:0:2::/64 or 2600:2d00:0:3::/64.
+      If you do not specify the next hop addresses, Google Cloud automatically
+      assigns unused addresses from the 2600:2d00:0:2::/64 or 2600:2d00:0:3::/64 range for you.
+    default_from_api: true
```

Comment:
Why should this default from API if it must be set when enableIpv6 is set?

Reply:
It can only be set when enableipv6 is set.
are there other options to approach this? 
thanks